### PR TITLE
Athena to packages

### PR DESCRIPTION
**Pull request by fiskus**
Sample table view is in `/b/fiskus-sandbox-dev/queries/athena/fiskus-test/db3f6700-906c-434d-9119-ee18b4880e0b` on searchminimal stack

Screenshot from 2022-08-24 13-55-28

Screenshot from 2022-08-24 13-55-38

- [x] Documentation
- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  return row.reduce((acc, value, index) => {
+    if (!head[index]) return acc
+    return {
+      ...acc,
+      [head[index]]: value,
+    }
+  }, {} as ManifestEntryStringified)
+}
+
+function parseManifenstEntryStringified(entry: ManifestEntryStringified): {
```
Comment (nl0) [inline catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx:50]:
```suggestion
function parseManifestEntryStringified(entry: ManifestEntryStringified): {
```

File: catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      <a href="https://docs.quiltdata.com/advanced/athena" target="_blank">
+        <M.IconButton>
+          <M.Icon>help_outline</M.Icon>
+        </M.IconButton>
+      </a>
+    </M.Tooltip>
+  )
+}
+
+function areQueryResultsContainMainefstEntries(
```
Comment (nl0) [inline catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx:26]:
```suggestion
function doQueryResultsContainManifestEntries(
```

**Review by nl0 — APPROVED** [review]
looks good sans some typos

File: catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ function SeeDocsForCreatingPackage() {
   )
 }
 
-function areQueryResultsContainMainefstEntries(
+function doQueryResultsContainMainefstEntries(
```
[the remark this answers is not in the document]
Comment (fiskus) [reply catalog/app/containers/Bucket/Queries/Athena/CreatePackage.tsx:26]: Sorry :|

**Review by nl0 — APPROVED** [review]

**Pull request merged**